docs/companion.md

```diff
@@ -490,28 +502,6 @@ Controls how big the uploaded chunks are for AWS S3 Multipart and Tus.
 Smaller values lead to more overhead, but larger values lead to slower retries in case of bad network connections.
 Passed to tus-js-client [`chunkSize`](https://github.com/tus/tus-js-client/blob/master/docs/api.md#chunksize) as well as [AWS S3 Multipart](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html) `partSize`.
 
-### Standalone only options
-
-These environment variables apply only to Companion running in [standalone mode](#standalone).
-
-#### `COMPANION_COOKIE_DOMAIN`
-
-Allows you to customize the domain of the cookies created for Express sessions.
-
-#### `COMPANION_ONEDRIVE_DOMAIN_VALIDATION`
-
-Settings this variable to `true` enables a route that can be used to validate your app with OneDrive. Only set this if you are sure that you are also setting the correct value for `COMPANION_ONEDRIVE_KEY` (i.e application ID - **not** the secret!)
-
-#### `COMPANION_HIDE_WELCOME`
-
-Setting this to `true` disables the welcome message shown at `/`.
-
-### Options `_FILE` suffix
-
-Many environment variables can alternatively be specified using a `_FILE` suffix. In this case the secret will be loaded from a file instead of being specified directly on the command line. This can be useful to prevent accidentally leaking secrets.
-
-Here is a list of the suffixed options: `COMPANION_GOOGLE_SECRET_FILE`, `COMPANION_DROPBOX_SECRET_FILE`, `COMPANION_BOX_SECRET_FILE`, `COMPANION_INSTAGRAM_SECRET_FILE`, `COMPANION_FACEBOOK_SECRET_FILE`, `COMPANION_ONEDRIVE_SECRET_FILE`, `COMPANION_ZOOM_SECRET_FILE`, `COMPANION_ZOOM_VERIFICATION_TOKEN`, `COMPANION_AWS_SECRET_FILE`, `COMPANION_SECRET_FILE`, `COMPANION_PREAUTH_SECRET_FILE`
```

Comment:
Shouldn't we document this somewhere?

Reply:
They were already there, I just de-duplicated them

<img width="993" alt="Screenshot 2022-12-27 at 12 40 43" src="https://user-images.githubusercontent.com/9060226/209661466-996af42c-7f8c-432f-a9a5-15bf42029fa9.png">

Reply:
I believe some of them are missing

Reply:
Added some missing ones. I still have to document `COMPANION_AWS_SECRET_FILE` and `COMPANION_SECRET_FILE` but I believe they need more explanation because unlike the others, it seems to mean multiple vars per file? How does that need to look then?

Reply:
Where do you see that they mean multiple vars per file?

Reply:
somehow thought it would be a replacement for the S3 vars, such as the key. But it's only the secret there too of course. Added the last ones